README.vmware_desktop

```diff
@@ -0,0 +1,38 @@
+vmware_desktop provider setup on Macbook M Series aarch64 processor
+===================================================================
+
+ 1. Install HomeBrew
+    `xcode-select --install`
+    `/usr/bin/ruby -e "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install)"`
+
+ 2. Install Vagrant
+    `brew install --cask vagrant`
+
+ 3. Install Vmware Fusion
+
+    Registering an account with Broadcom is a pre-requisite.
+
+    Download Vmware Fusion from https://support.broadcom.com/group/ecx/productdownloads?subfamily=VMware+Fusion
+
+    Note: 'Fusion' is the MAC version of Vmware Desktop. The following may
+    also work with Vmware Desktop on Linux but has not been verified.
+
+    Install vmware utility:
+    `brew install --cask vagrant-vmware-utility`
+
+    Install the vmware-desktop vagrant plugin:
+    `sudo vagrant plugin install vagrant-vmware-desktop`
+
+4. Create and source a localrc file, then `vagrant up`, as documented in README file.
```

Comment:
do you have to set `VAGRANT_DEFAULT_PROVIDER`?

Reply:
yes but I do still wonder if that wouldn't be better off in the README, e.g. a new section that covers non-default providers and points to the other README.<provider> for tips on installing them.

Reply:
Yeah that would make thing in this MR, basically instead of "assume virtualbox and vagrant" https://github.com/nvidia/vagrant-swift-all-in-one?tab=readme-ov-file#vagrant-swift-all-in-one 

... we just say "first you need to pick provider" and then I guess...

... like a table of: 

provider => supported/tested ubuntu image version(s), host-os and arch?

where each provider is a link to it's README.provider?